### PR TITLE
ci(releasenotes): lint reno files

### PR DESCRIPTION
**Pull request by sileht**
Change-Id: I4a1131cc4c1cef5d591b7c7aac2bf7c1119a5028

### CODE REVIEW

File: .github/workflows/ci-pep8.yaml [hunk 1]
```diff
@@ -14,6 +14,8 @@ jobs:
     steps:
       - name: Checkout 🛎️
         uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0
```
Comment (jd) [inline .github/workflows/ci-pep8.yaml:18]: sounds unrelated to the pr?

**Review by jd — CHANGES_REQUESTED** [review]
might be good splitting it

File: .github/workflows/ci-pep8.yaml (hunk 1 above)
Comment (sileht) [reply .github/workflows/ci-pep8.yaml:18]: Reno needs full git clone to work

Comment (jd) [reply .github/workflows/ci-pep8.yaml:18]: ok, would have been great to write this in the commit message :)

**Review by jd — APPROVED** [review]

**Pull request merged**